mmeds/validate.py

```diff
@@ -129,33 +128,36 @@ def __init__(self, file_fp, study_name, metadata_type, subject_ids, subject_type
         # used when validating specimen metadata
         self.subject_ids = subject_ids
 
+        self.user = user
+        self.testing = testing
+
     def check_number_column(self, column):
         """ Check for mixed types and values outside two standard deviations. """
-        Logger.debug("check number column")
+        #Logger.debug("check number column")
         filtered = [self.col_type(x) for x in column.tolist() if is_numeric(x)]
         Logger.debug("Filtered")
         Logger.debug(filtered)
         stddev = std(filtered)
         avg = mean(filtered)
         for i, cell in enumerate(filtered):
-            Logger.debug(f"{i}: {cell}")
+            # Logger.debug(f"{i}: {cell}")
             if (cell > avg + (2 * stddev) or cell < avg - (2 * stddev)):
-                Logger.debug("Outside stddev")
+                #Logger.debug("Outside stddev")
                 text = '{}\t{}\tStdDev Warning: Value {} outside of two standard deviations of mean in column {}'
                 self.warnings.append(text.format(i, self.col_index, cell, self.col_index))
-        Logger.debug("Finished check number column")
+        #Logger.debug("Finished check number column")
 
     def check_string_column(self, column):
         """ Check for categorical data. """
-        Logger.debug("In check string column")
+        #Logger.debug("In check string column")
         counts = column.value_counts()
         stddev = std(counts.values)
         avg = mean(counts.values)
-        Logger.debug('Got counts')
+        #Logger.debug('Got counts')
```

Comment:
commented out logger statements should be removed

Reply:
gonna put them back in. removed them in the process of fixing some tests, they got in the way of seeing what was wrong.